packages/wrangler/src/__tests__/workflows.test.ts

```diff
@@ -0,0 +1,426 @@
+				version_id: "c",
+				status: "running",
+			},
+			{
+				id: "bar",
+				created_on: "2022-01-01T00:00:00Z",
+				modified_on: "2022-01-01T00:00:00Z",
+				workflow_id: "b",
+				version_id: "c",
+				status: "running",
+			},
+		];
+
+		it("should get and pause the bar instance given a name", async () => {
+			writeWranglerToml();
+			await mockGetInstances(mockInstances);
+			await mockPatchRequest("bar");
+
+			await runWrangler(`workflows instances pause some-workflow bar`);
+			expect(std.info).toMatchInlineSnapshot(
+				`"⏸️ The instance \\"bar\\" from some-workflow was paused successfully"`
+			);
+		});
+	});
+
+	describe("instances resume", () => {
+		const mockInstances: Instance[] = [
+			{
+				id: "foo",
+				created_on: "2021-01-01T00:00:00Z",
+				modified_on: "2021-01-01T00:00:00Z",
+				workflow_id: "b",
+				version_id: "c",
+				status: "running",
+			},
+			{
+				id: "bar",
+				created_on: "2022-01-01T00:00:00Z",
+				modified_on: "2022-01-01T00:00:00Z",
+				workflow_id: "b",
+				version_id: "c",
+				status: "paused",
+			},
+		];
+
+		it("should get and resume the bar instance given a name", async () => {
+			writeWranglerToml();
+			await mockGetInstances(mockInstances);
+			await mockPatchRequest("bar");
+
+			await runWrangler(`workflows instances resume some-workflow bar`);
+			expect(std.info).toMatchInlineSnapshot(
+				`"🔄 The instance \\"bar\\" from some-workflow was resumed successfully"`
+			);
+		});
+	});
+
+	describe("instances terminate", () => {
+		const mockInstances: Instance[] = [
+			{
+				id: "foo",
+				created_on: "2021-01-01T00:00:00Z",
+				modified_on: "2021-01-01T00:00:00Z",
+				workflow_id: "b",
+				version_id: "c",
+				status: "running",
+			},
+			{
+				id: "bar",
+				created_on: "2022-01-01T00:00:00Z",
+				modified_on: "2022-01-01T00:00:00Z",
+				workflow_id: "b",
+				version_id: "c",
+				status: "running",
+			},
+		];
+
+		it("should get and terminate the bar instance given a name", async () => {
+			writeWranglerToml();
+			await mockGetInstances(mockInstances);
+			await mockPatchRequest("bar");
+
+			await runWrangler(`workflows instances terminate some-workflow bar`);
+			expect(std.info).toMatchInlineSnapshot(
+				`"🥷 The instance \\"bar\\" from some-workflow was terminated successfully"`
+			);
+		});
+	});
+
+	describe("trigger", () => {
+		const mockTriggerWorkflow = async () => {
+			msw.use(
+				http.post(
+					`*/accounts/:accountId/workflows/some-workflow/instances`,
+					async () => {
+						return HttpResponse.json({
+							success: true,
+							errors: [],
+							messages: [],
+							result: {},
+						});
+					},
+					{ once: true }
+				)
+			);
+		};
+
+		it("should trigger a workflow given a name", async () => {
+			writeWranglerToml();
+			await mockTriggerWorkflow();
+
+			await runWrangler(`workflows trigger some-workflow`);
+			expect(std.info).toMatchInlineSnapshot(
+				`"🚀 Workflow instance \\"undefined\\" has been queued successfully"`
```

Comment:
Can we provide a mock response result so the log message will reflect how it uses the response result (i.e. `response.id`) instead of printing `undefined` here?